docs/docs/debug/upgrade.md

```diff
@@ -0,0 +1,324 @@
+---
+id: upgrade
+title: Ory Kratos Upgrade Guide
+---
+
+This guide covers essentials to consider when upgrading Ory Kratos, please also
+refer to [UPGRADE.md](https://github.com/ory/kratos/blob/master/UPGRADE.md).
+
+Generally after upgrading just run `kratos migrate`.
```

Comment:
Please always warn before upgrades:

1. Explain the process
2. Make clear that they should run it against a test env first
3. Make a backup
4. Run the process itself

Reply:
Please also look at `kratos help migrate sql`, I am sure you can use some info from there :) You can also link to the command in the CLI reference.